src/Evolution/Systems/Cce/Actions/InitializeCharacteristicEvolution.hpp

```diff
@@ -77,15 +78,19 @@ namespace Actions {
  *  - `Tags::Variables<metavariables::cce_swsh_derivative_tags>`
  *  - `Spectral::Swsh::Tags::NumberOfRadialPoints`
  *  - `Tags::EndTime`
+ *  - `Spectral::Swsh::Tags::SwshInterpolator< Tags::CauchyAngularCoords>`
  * - Removes: nothing
  */
 struct InitializeCharacteristicEvolution {
   using initialization_tags =
-      tmpl::list<InitializationTags::LMax,
-                 InitializationTags::NumberOfRadialPoints,
-                 InitializationTags::StartTime, InitializationTags::EndTime,
+      tmpl::list<InitializationTags::StartTime, InitializationTags::EndTime,
                  InitializationTags::TargetStepSize>;
-  using const_global_cache_tags = tmpl::list<::Tags::TimeStepper<TimeStepper>>;
+  using const_global_cache_tags =
+      tmpl::list<::Tags::TimeStepper<TimeStepper>, Spectral::Swsh::Tags::LMax,
+                 Spectral::Swsh::Tags::NumberOfRadialPoints,
+                 InitializationTags::FilterLMax,
```

Comment:
put these new tags in a `using const_global_cache_tags` type alias in the action that uses them, not the initialization action.